src/main/java/com/cleanroommc/groovyscript/sandbox/GroovyScriptSandbox.java

```diff
@@ -162,6 +165,13 @@ public void run(LoadStage currentLoadStage) {
         }
     }
 
+    @Override
+    protected void runScript(Script script) {
+        securityManager.install();
+        super.runScript(script);
+        securityManager.uninstall();
```

Comment:
If the script accesses outside the instance dir in closures such as event listeners, it can still be bypassed no?

Reply:
true